internals-js/src/error.ts

```diff
@@ -325,6 +339,17 @@ const INVALID_LINK_DIRECTIVE_USAGE = makeCodeDefinition(
   'An application of the @link directive is invalid/does not respect the specification.'
 );
 
+const LINK_IMPORT_NAME_MISMATCH = makeCodeDefinition(
+  'LINK_IMPORT_NAME_MISMATCH',
+  'The import name for a merged directive (as declared by the relevant `@link(import:)` argument) is inconsistent between subgraphs.'
+);
+
+const REFERENCED_INACCESSIBLE = makeCodeDefinition(
+  'REFERENCED_INACCESSIBLE',
+  'An element is marked as @inaccessible but is referenced by a non-inacessible elemnent (at least in post-merging of the subgraphs)'
```

Comment:
Agreed about the "post-merging" parts that it's not super clear. I ended up removing it because 1) I wasn't sure either how to say that better and 2) that error can also be thrown by non-merge related calls to `toAPISchema` so it's probably not worth trying to be overly precise after all.